src/main/java/com/exadel/backendservice/entity/EventEntity.java

```diff
@@ -0,0 +1,57 @@
+package com.exadel.backendservice.entity;
+
+import com.exadel.backendservice.model.EventType;
+import lombok.Data;
+import lombok.EqualsAndHashCode;
+import lombok.NoArgsConstructor;
+import lombok.ToString;
+import org.hibernate.annotations.Type;
+
+import javax.persistence.*;
+import java.sql.Timestamp;
+import java.util.HashSet;
+import java.util.Set;
+
+@Entity
+@Table(name = "event_table")
+@Data
+@NoArgsConstructor
+public class EventEntity {
+
+    @Id
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
+    private Integer id;
+
+    @Column(length = 64, nullable = false)
+    private String name;
+
+    @Column(length = 256, nullable = false)
+    private String description;
+
+    @Column(name = "start_date")
+    private Timestamp startDate;
+
+    @Column(name = "end_date")
+    private Timestamp endDate;
```

Comment:
please use LocalDateTime instead of Timestamp

Reply:
corrected